Source/JavaScriptCore/Scripts/postprocess-header-rule

```diff
@@ -91,14 +91,17 @@ function rewrite_headers () {
     fi
 
     SED_OPTIONS+=(${OTHER_SED_OPTIONS[*]})
+fi
 
-    sed -E "${SED_OPTIONS[@]}" "${SOURCE_FILE}" > "${WORK_FILE}" || exit $?
-    cmp -s "${WORK_FILE}" "${DEST_FILE}" && rm -f "${WORK_FILE}" || mv "${WORK_FILE}" "${DEST_FILE}"
-    [[ "${SOURCE_FILE}" -nt "${DEST_FILE}" ]] && touch "${DEST_FILE}" || true
-}
-
-DEFINITIONS_PATH=usr/local/include/WebKitAdditions/Scripts/postprocess-framework-headers-definitions
+UNIFDEF_OPTIONS=(-Bt)
```

Comment:
`-t` turns off some unneeded source parsing behavior. unifdef does not understand c++14 style single-quotes used as separators in integer literals, like
https://github.com/WebKit/WebKit/blob/f67bf0023bc8a15007490b887f367c001fbddf93/Source/JavaScriptCore/assembler/ARM64Assembler.h#L1600